sys/mathoid.js

```diff
@@ -103,6 +106,69 @@ MathoidService.prototype.checkInput = function(hyper, req) {
 };
 
 
+MathoidService.prototype._storeRenders = function(hyper, domain, hash, completeBody) {
+
+    var idx;
+    var len = FORMATS.length;
+    var reqs = new Array(len);
+
+    for (idx = 0; idx < len; idx++) {
```

Comment:
Minor: Since this is a small number of entries, FORMATS.forEach(..) should perform well enough, and has the benefit of being clearer.